extensions/cloud/portability-cloud-microsoft/src/main/java/org/dataportabilityproject/cloud/microsoft/cosmos/DataWrapper.java

```diff
@@ -0,0 +1,35 @@
+package org.dataportabilityproject.cloud.microsoft.cosmos;
+
+import com.microsoft.azure.storage.table.TableServiceEntity;
+
+/** Wraps Job data for serialization to Cosmos DB */
+public class DataWrapper extends TableServiceEntity {
+  private String serialized;
```

Comment:
Both of these can be final

Reply:
The fields can't because Azure needs to reflectively set them. The API has a bit of a strange design.